code/go/0chain.net/smartcontract/benchmark/main/cmd/mpt.go

```diff
@@ -199,19 +205,36 @@ func setUpMpt(
 
 	var eventDb *event.EventDb
 	if viper.GetBool(benchmark.EventDbEnabled) {
-		timer = time.Now()
+		timer := time.Now()
+
+		tick := func() (*event.EventDb, error) {
+			return event.NewEventDb(dbs.DbAccess{
```

Comment:
I am getting lots of warnings like
```
postgres_1   | 2022-06-02 16:26:46.104 UTC [73] ERROR:  duplicate key value violates unique constraint "idx_miners_miner_id"
```
Perhaps we could turn them off somehow?

Reply:
imo we better fix them, I couldn't find place where duplicates are added

Reply:
I suspect it might be something like [exist](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/dbs/event/miner.go#L229) methods. 

It is possible that someone decided it was quicker to do a query they expect to fail, then respond sensibly to the error. This might or might not be true, but also gives lots of annoying warnings.

Reply:
yep, we can address devops to tune postresss